go.mod

```diff
@@ -1,32 +1,36 @@
 module github.com/Simple-Tracker/qBittorrent-ClientBlocker
 
-go 1.20
+go 1.21.0
```

Comment:
To maintain compatibility, it is not recommended to upgrade the Go version. (Go 1.21 drops support for Windows 7, but there are no features that require urgent updates for now)
As an alternative, you can specify the go-toml version v2.2.2.

Reply:
Specify the version to `v2.2.2`.